File: JavaScript/1/1_4.js
function calculateCircleProperties(radius) {
  const PI = 3.14;
  const area = PI * radius * radius;
  const circumference = 2 * PI * radius;
  
  console.log("Area: " + area);
  console.log("Circumference: " + circumference);
}

calculateCircleProperties(5);

File: C#/1/1_4.cs
using System;

class Program
{
    static void Main()
    {
        CalculateCircle(5);
    }

    static void CalculateCircle(double radius)
    {
        double area = 3.14 * radius * radius;
        double circumference = 2 * 3.14 * radius;
        Console.WriteLine("Area: " + area);
        Console.WriteLine("Circumference: " + circumference);
    }
}
